packages/libs/eda/src/lib/workspace/DownloadTab/StudyCitation.tsx

```diff
@@ -0,0 +1,151 @@
+
+// Definitions
+import { DownloadTabStudyRelease } from './types';
+import { colors, Paragraph } from '@veupathdb/coreui';
+
+// Utils
+import {
+  writeTextToClipboard,
+  stripHTML,
+} from '@veupathdb/wdk-client/lib/Utils/DomUtils';
+import { safeHtml } from '@veupathdb/wdk-client/lib/Utils/ComponentUtils';
+import { makeStyles } from '@material-ui/core';
+
+export type CitationDetails = {
+  partialCitationData: {
+    studyContacts: string;
+    studyDisplayName: string;
+    projectDisplayName: string;
+    citationUrl: string;
+  };
+  release: DownloadTabStudyRelease;
+};
+
+export function getCitationString({
+  partialCitationData,
+  release,
+}: CitationDetails) {
+  const { studyContacts, studyDisplayName, projectDisplayName, citationUrl } =
+    partialCitationData;
+
+  /**
+   * In the event an error occurs while parsing or formatting the date, we fall back
+   * on the release date format provided from the backend
+   */
+  let citationDate;
+  try {
+    const parsedReleaseDate = parse(
+      release.date ?? '',
+      'yyyy-MMM-dd',
+      new Date()
+    );
+    citationDate = format(parsedReleaseDate, 'dd MMMM yyyy');
+  } catch {
+    citationDate = release.date ?? 'date unknown';
+  }
+  return `${studyContacts}. Dataset: ${studyDisplayName}. ${projectDisplayName}. ${citationDate}, Release ${release.releaseNumber} (${citationUrl})`;
+}
+
+const useStyles = makeStyles(() => ({
+  tooltip: {},
+}));
+
+export default function StudyCitation({
+  partialCitationData,
+  release,
+}: CitationDetails) {
+  const [hoveredState, setHoveredState] = useState<boolean>(false);
+  const [tooltipPosition, setTooltipPosition] = useState({
+    pageX: 0,
+    pageY: 0,
+  });
+  const classes = useStyles();
+
+  const citation = getCitationString({
+    partialCitationData,
+    release,
+  });
+
+  const copyCitation = () => {
+    writeTextToClipboard(stripHTML(citation));
+  };
+
+  const handleMouseMove: MouseEventHandler<HTMLDivElement> = (event) => {
+    const { pageX, pageY } = event;
+    setTooltipPosition({ pageX, pageY });
+  };
+
+  const computeStyleFn = (data: any) => {
+    return {
+      ...data,
+      styles: {
+        ...data.styles,
+        left: `${tooltipPosition.pageX + 5}px`,
+        top: `${tooltipPosition.pageY + 5}px`,
+      },
+    };
+  };
+
+  return (
+    <Paragraph
+      color={colors.gray[600]}
+      styleOverrides={{ margin: 0 }}
+      textSize="medium"
+    >
+      <span style={{ fontWeight: 500 }}>Dataset Citation: </span>
+      <Tooltip
+        title="Copy citation to clipboard"
+        onMouseMove={handleMouseMove}
+        classes={{ tooltip: classes.tooltip }}
+        PopperProps={{
+          modifiers: {
+            computeStyle: {
+              fn: computeStyleFn,
+              gpuAcceleration: true,
+            },
+          },
+        }}
+      >
+        <span
+          onMouseOver={() => (hoveredState ? undefined : setHoveredState(true))}
```

Comment:
It felt a bit odd to have hovering/copying over both the citation string and the button. Can we do just one or the other? I'm partial to the copy button strategy :)

Reply:
I can revert these changes and just do the copy button. The `followCursor` attribute is available in newer MUI versions, so at least it was fun implementing this logic if we are to remove it! 😂